charts/dependabot-gitlab/README.md

```diff
@@ -119,6 +119,13 @@ By default chart installs instance of [redis](https://github.com/bitnami/charts/
 | worker.probePort | int | `7433` |  |
 | worker.replicaCount | int | `1` | Worker container replicas count |
 | worker.resources | object | `{}` | Worker container resource definitions |
+| worker.serviceMonitor.additionalLabels | object | `{}` |  |
+| worker.serviceMonitor.enabled | bool | `true` |  |
+| worker.serviceMonitor.metricRelabelings | list | `[]` |  |
+| worker.serviceMonitor.namespace | string | `""` |  |
+| worker.serviceMonitor.namespaceSelector | object | `{}` |  |
```

Comment:
I believe this should be set to `..Release.namespace` in the template and should not be configurable